app/api/v1/endpoints/tenants.py

```
@@ -120,6 +129,109 @@ async def list_tenants(
+        raise HTTPException(status_code=404, detail="Tenant not found")
+    latest = await history_repo.get_latest_for_tenant(tenant_id)
+    return TenantIntegrityStatus(
+        profile=tenant.integrity_profile,
+        last_changed_at=latest.changed_at if latest else None,
+        cooling_off_ends_at=latest.cooling_off_ends_at if latest else None,
+    )
+
+
+@router.put("/integrity", response_model=TenantIntegrityStatus)
+async def update_tenant_integrity(
+    request: Request,
```

Ruff (0.15.5)

[warning] Unused function argument: `request`

(ARG001)